src/box/lua/misc.cc

```diff
@@ -61,6 +63,29 @@ lbox_encode_tuple_on_gc(lua_State *L, int idx, size_t *p_len)
 	return (char *) region_join_xc(gc, *p_len);
 }
 
+API_EXPORT const char *
+position_pack_on_gc(struct position *pos, uint32_t *size)
+{
+	assert(pos != NULL);
+	assert(size != NULL);
+	assert(cord() != NULL);
+
+	*size = position_pack_size(pos);
+	char *buf = NULL;
+	struct errinj *inj = errinj(ERRINJ_POSITION_PACK_ON_GC_ALLOC,
+				    ERRINJ_BOOL);
+	if (inj == NULL || inj->bparam == false)
+		buf = (char *)region_alloc(&fiber()->gc, *size);
```

Comment:
Looks like you don't clean up the region after usage. The fiber region isn't cleaned up automatically - it should be cleaned up explicitly with `region_truncate`.

Reply:
done (in LuaC too)